treeherder/extract/extract_alerts.py

```diff
@@ -0,0 +1,139 @@
+import logging
+
+from redis import Redis
+
+from jx_bigquery import bigquery
+from jx_mysql.mysql import (MySQL,
+                            quote_value)
+from jx_mysql.mysql_snowflake_extractor import MySqlSnowflakeExtractor
+from mo_files import File
+from mo_json import (json2value,
+                     value2json)
+from mo_logs import (Log,
+                     constants,
+                     startup)
+from mo_sql import SQL
+from mo_times import (DAY,
+                      Timer)
+from mo_times.dates import (Date,
+                            parse)
+from treeherder.extract import VENDOR_PATH
+
+CONFIG_FILE = (File.new_instance(__file__).parent / "extract_alerts.json").abspath
+
+_keep_import = VENDOR_PATH
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+class ExtractAlerts:
+    def run(self, force=False, restart=False, merge=False):
+        # SETUP LOGGING
+        settings = startup.read_settings(filename=CONFIG_FILE)
```

Comment:
Can we just use a standard Django logger here?  I'm not sure how this will play with Papertrail (or whatever logging system we migrate to at some point.). This would also ensure that it honors the visibility settings we choose in the Django settings.  You already have one at your disposal!  `logger`. :)

Reply:
@camd  The libraries are using a global structured logger (intended for inserting logs into data warehouses).  This line tells that structured logger to emit logs to `logging.getLogger("extract_alerts")`

Reply:
@camd If the above is unacceptable then what about

    Log.use_logger(logging.getLogger(__name__))

?

Reply:
I'd say the name isn't important.  So either `extract_alerts` or `__name` are fine by me.  :)  As long as it goes through the standard logging.  Even if your `Log` object adds other stuff do it, that works for me.